authors/apps/articles/models.py

```diff
@@ -16,22 +14,17 @@
 
 class Articles(models.Model):
     """ Model for all articles """
-    slug = models.SlugField(max_length=250, default='non')
-    title = models.CharField(max_length=50, default='non')
-    description = models.CharField(max_length=250, default='non')
-    body = models.CharField(max_length=550, default='non')
-    tagList = models.CharField(max_length=50,
-                               default='non')  # ["dragons", "training"],
+    slug = models.SlugField(unique=True, max_length=250)
+    title = models.CharField(max_length=250)
+    description = models.CharField(max_length=350)
+    body = models.TextField()
+    tagList = models.CharField(max_length=200)
     createdAt = models.DateTimeField(auto_now_add=True)
     updatedAt = models.DateTimeField(auto_now=True)
     favorited = models.BooleanField(default=False)
```

Comment:
The `favorited` and `favoritesCount` fields don't need to be defined here, in `Articles`, because they already exist in the `Favorites` app. The serializer can return these, retrieving them from the `Favorites` app.
Please remove these, to remove duplication of data in the db.

Reply:
Well noted.

Reply:
@dmithamo1 I would request that to be done in a different pr(bug fix as at the moment it will break a number of tests thus affecting the merging ot the rest of the PRs)

Reply:
Makes sense. On it.